packages/config/config/devices/0x0346/glass_break_sensor.json

```diff
@@ -0,0 +1,252 @@
+// TODO: This file contains a placeholder for the ProductType or Product ID (0x9999) that must be corrected. 
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "GLASS BREAK SENSOR",
+	"description": "Glass Break Sensor",
+	"devices": [
+		{
+			"productType": "0x9999",
+			"productId": "0x0a01",
```

Comment:
```suggestion
			"productId": "0x0301",
```

Reply:
I suppose the ZWA listing is incorrect here based on the debug info from Z-Wave JS UI.